### ISSUE TITLE
Allow client to do updating

### ISSUE THREAD

**Issue by WickedShell** [jdolan/quetoo#46]
The client should be able to from a menu run the rsync update and pull all the update data. This would vastly improve user experience, and make it smoother.

Once this is done, it's worth considering compiling enough assets in (mostly a(the?) default font), so that a simple GUI saying "No game data, do you wish to update?" Could be displayed. That's probably a separate task issue though, just where I'd like to take this once this is done.
